src/content/docs/infrastructure/host-integrations/host-integrations-list/apache-traffic-server-monitoring-integration.mdx

```diff
@@ -29,34 +29,78 @@ You can install the infrastructure agent two different ways:
 
 * Our [guided install](https://one.newrelic.com/nr1-core?state=4f81feab-35f7-e97e-9903-52510f8542bd) is a CLI tool that inspects your system and installs the infrastructure agent alongside the application monitoring agent that best works for your system. To learn more about how our guided install works, check out our [Guided install overview](/docs/infrastructure/host-integrations/installation/new-relic-guided-install-overview).
 * If you'd rather install our infrastructure agent manually, you can follow a tutorial for manual installation for [Linux](/docs/infrastructure/install-infrastructure-agent/linux-installation/install-infrastructure-monitoring-agent-linux), [Windows](/docs/infrastructure/install-infrastructure-agent/windows-installation/install-infrastructure-monitoring-agent-windows/), or [macOS](/docs/infrastructure/install-infrastructure-agent/macos-installation/install-infrastructure-monitoring-agent-macos/).
+
+## Install Apache New Relic Agent
+Our Apache integrations collects network metrics like TCP connections, DNS lookup, HTTPS, SSL, and server and worker statuses.
+To install the integration, follow the steps in our [guided install](https://one.newrelic.com/launcher/nr1-core.explorer?pane=eyJuZXJkbGV0SWQiOiJucjEtY29yZS5saXN0aW5nIn0=&cards%5B0%5D=eyJuZXJkbGV0SWQiOiJucjEtaW5zdGFsbC1uZXdyZWxpYy5ucjEtaW5zdGFsbC1uZXdyZWxpYyIsInBhdGgiOiJvaGkiLCJyZWNpcGVOYW1lIjoiYXBhY2hlLW9wZW4tc291cmNlLWludGVncmF0aW9uIiwiYWN0aXZlQ29tcG9uZW50IjoiVlRTT0NvbW1hbmQiLCJhY3RpdmVFbnZpcm9ubWVudCI6ImFwYWNoZS1vcGVuLXNvdXJjZS1pbnRlZ3JhdGlvbiJ9).
+If you need to install the integration in a different way, see [Apache monitoring integration](https://docs.newrelic.com/docs/infrastructure/host-integrations/host-integrations-list/apache-monitoring-integration/).
+
+## Expose your metrics
+After successful installation, Apache Traffic Server has to be started successfully, then it should start running on a `your_ip` with port `8080`. You need to configure the metrics exposed stats.
+
+Goto the `plugin.config` file.
+  ```shell
+  sudo nano /etc/trafficserver/plugin.config
+  ```
+Add the following line `stats_over_http.so` in your `plugin.config` file.
+
+Make sure your metrics are exposed on the below URLs.
+Metrics of name node: `http://YOUR_IP:8080/_stats`
+
 ## Configure NRI-Flex for Apache Traffic Server [#integrate]
 
 Flex comes bundled with the New Relic infrastructure agent. To create a flex configuration file follow these steps:
 
-1. Create a file named `nri-flex-apache-traffic-server-config.yml` in this path:
+1. Creates files with  named `nri-flex-apache-traffic-server-config.yml`, `nri-flex-ats-cache-config.yml` and `nri-flex-ats-http-config.yml` in this path:
```

Comment:
```suggestion
1. Creates files named `nri-flex-apache-traffic-server-config.yml`, `nri-flex-ats-cache-config.yml`, and `nri-flex-ats-http-config.yml` in this path:
```

Reply:
Hi @ally-sassman,
As per your suggestion, I updated the changes. Kindly review it.

Thanks.